### PR TITLE
Add support for decimal precision and range options

### PR DESCRIPTION
**Pull request by GrzesiekGdn**
Example of column definition:

```
balance: { type: 'decimal', precision: "9", scale: "2", notNull: true }
```

issue #3

### CODE REVIEW

**Comment by pc-jedi** [comment]
+1

**Comment by wzrdtales** [comment]
@GrzesiekGdn Thank you for your contribution!

Please take a look at my remarks.

This is currently also Missing tests.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "db-migrate-mysql",
-  "version": "1.1.6",
+  "version": "1.1.7",
```
Comment (wzrdtales) [inline package.json]: Remove this please, you never should ever alter the package.json, unless you install a new package, which also would need to be justified.

**Comment by GrzesiekGdn** [comment]
Ok.
G.

**Pull request closed without merging**